CM7/Core/Inc/main.h

```diff
@@ -108,6 +108,16 @@ void Error_Handler(void);
 #define NRF_CE_GPIO_Port GPIOC
 #define MOTOR1_PWM_Pin GPIO_PIN_8
 #define MOTOR1_PWM_GPIO_Port GPIOA
+#define SPI_CS_OUTPUT_Pin GPIO_PIN_3

```

Comment:
släng in dom här i READMEn också, det finns tabeller för CN7/CN10 om dom hör till någon av dom